extensions/VideoManager/VideoManager.fs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright 2018 Fabulous contributors. See LICENSE.md for license.
+namespace Fabulous.DynamicViews
+
+[<AutoOpen>]
+module VideoManagerExtension =
+
+    open Fabulous.DynamicViews
+    open MediaManager.Forms
+    open MediaManager.Video
+
+    //// Define keys for the possible attributes
+    let SourceAttribKey = AttributeKey<_> "VideoManager_Source"
+    let VideoAspectAttribKey = AttributeKey<_> "VideoManager_VideoAspect"
+    let ShowControlsAttribKey = AttributeKey<_> "VideoManager_ShowControls"
+
+    type Fabulous.DynamicViews.View with
+        /// Describes a VideoView in the view
+        static member inline VideoView
+            (source: obj, ?videoAspect: VideoAspectMode , ?ShowControls: bool,
```

Comment:
Are you sure for the source being optional? What is the policy here?
I'm going to do it whatever the response is if there is a doubt. :)

Reply:
The main reason is to avoid needing `null`.

Even if your player will eventually play something, you might not know at initialization time the source of the video.
Not setting it optional would require you to do `View.VideoView(null)` which would be weird, given we prevent the use of null everywhere else.

With an optional source, this would be possible instead:
```fsharp
type Model = { Source : string option }
View.VideoView(?source=model.Source)
```